nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Understanding internal transactions and delivery guarantees
+reviewed: 2016-04-20
+tags:
+- Azure
+- Cloud
+- Transactions
+---
+
+## Understanding internal transactions and delivery guarantees
+
+### Versions 6 and below
```

Comment:
@weralabaj is that the guideline to use [partials](https://github.com/Particular/docs.particular.net/#markdown-partials) whenever there's a section that is different between versions?